es-space-mgmt/main/default/lwc/reservationDetailForm/__tests__/reservationDetailForm.test.js

```diff
@@ -0,0 +1,244 @@
+import { createElement } from 'lwc';
+import getMarketsByState from '@salesforce/apex/marketServices.getMarketsByState';
+import { registerApexTestWireAdapter } from '@salesforce/sfdx-lwc-jest';
+import ReservationDetailForm from 'c/reservationDetailForm';
+
+// Register as Apex wire adapter. Some tests verify that provisioned values trigger desired behavior.
+const getMarketsByStateAdapter = registerApexTestWireAdapter(getMarketsByState);
+
+// Realistic data with a list of markets
+const mockMarketsByStateRecords = require('./data/getMarketsByState.json');
+
+// Realistic data with a list of markets returning 0 rows
+const mockMarketsByStateNoRecords = require('./data/getMarketsByStateNoRecords.json');
+
+// Constants for city, state and Start
+const CITY = 'San Francisco';
+const STATE = 'CA';
+const START_DATE = '1/1/2020';
+
+const CITY_OPTIONS = [
+    { value: '0031700000pJRRSAA4', label: 'San Francisco' },
+    { value: '0031700000pJRRSAA5', label: 'San Ramon' }
+];
+
+describe('c-reservation-detail-form', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+        // Prevent data saved on mocks from leaking between tests
+        jest.clearAllMocks();
+    });
+
+    it('shows error panel element', () => {
```

Comment:
The description should be more descriptive, for example `shows the error panel element when the Apex wire adapter getMarketsByState` (<= this would be the ultra long version).

Reply:
fixed!